### PR TITLE
Improve usage docs

### PR DESCRIPTION
**Pull request by VineethReddy02**
Signed-off-by: Vineeth Pothulapati <vineethpothulapati@outlook.com>

### CODE REVIEW

File: docs/cli-usage.md
Comment (blagojts) [inline docs/cli-usage.md:48]: I would put Grafana at the bottom

File: docs/cli-usage.md
Comment (blagojts) [inline docs/cli-usage.md:47]: Maybe add a sentance here that tells people that the values.yml file is what they would use with manual helm deployes and where and how they can generate the default file

File: docs/cli-usage.md
Comment (blagojts) [inline docs/cli-usage.md:50]:
```suggestion
tobs CLI offers a grafana cmd to interact with grafana with ease. 
Actions such as: get grafana instance password, change grafana instance password and port forwarding to locally interact with grafana. 
Usually to perform these actions we need to view the grafana K8s Secret & decode it from base64 and changing password needs Secret modification. All these actions will be performed by tobs.
```

File: docs/cli-usage.md
Comment (blagojts) [inline docs/cli-usage.md:76]: We should upload the photos to be part of the repo

File: docs/cli-usage.md
Comment (blagojts) [inline docs/cli-usage.md:249]:
```suggestion
### Actions for Volumes
```

**Review by blagojts — APPROVED** [review]
I like the new doc, left some small suggestions throughout, but am approving the PR so you don't wait for another review after the changes are in

File: docs/cli-usage.md
Comment (VineethReddy02) [reply docs/cli-usage.md:47]: The default file exists in `chart/values.yaml` right? I am not sure how can one generate values.yaml file?

**Pull request merged**
